gulp/tasks/test.js

```diff
@@ -2,11 +2,14 @@
 
 var gulp = require( 'gulp' );
 var $ = require( 'gulp-load-plugins' )();
-var protractor = require( 'gulp-angular-protractor' );
-var exec = require( 'child_process' ).exec;
+var childProcess = require( 'child_process' );
+var exec = childProcess.exec;
+var spawn = childProcess.spawn;
 var config = require( '../config' ).test;
+var minimist = require( 'minimist' );
+var path = require( 'path' );
 
-gulp.task( 'test:unit', function( cb ) {
+gulp.task( 'test:unit:js', function( cb ) {
```

Comment:
Agreed, I typed `gulp test:unit:scripts` last night. Let's update to `scripts` throughout, because we're only testing site scripts, not build JS.